README.md

```diff
@@ -154,6 +154,12 @@ infos:
   description: "A list of infos of the current page. Deprecated from ECL 2.30."
 ```
 
+In addition, the customization that allowed the content language switcher to be rendered inside the page header
+has been deprecated in favor of the new [ECL composition](https://europa-component-library.netlify.app/playground/ec/?path=/story/compositions-etranslation--default).
```

Comment:
Seems like link is broken.
```suggestion
has been deprecated in favor of the new [ECL composition](https://v3--europa-component-library.netlify.app/playground/ec/?path=/story/compositions-etranslation--default).
```

Reply:
It will not be when they release the actual version 3. the v3-- is a temporary URL.